### ISSUE TITLE
helm_remote: remove and pull chart each time

### ISSUE THREAD

**Issue by Tchoupinax** [tilt-dev/tilt-extensions#60]
Hello,

I worder about the installation of each chart. For *each* `helm_remote`, the command makes these instructions:

```python
# ======== Perform Installation
local('rm -rf %s' % pull_target)
local(pull_command)
install_crds(chart, chart_target)
``` 
So it will download each time the chart and remove the previous. Moreover, in my case i have a uniq chart for several service (called A, B, C for the exemple)

So, the script makes the following

```
downloading `myChart` and start it for service 'A'
removing `myChart`
downloading `myChart` and start it for service 'B'
removing `myChart`
downloading `myChart` and start it for service 'C'
removing `myChart`
```

As you can see, step are very useless because of removing and downloading the same chart.

**Comment by KOGI** [comment]
@Tchoupinax was this resolved with #59 ?

**Comment by Tchoupinax** [comment]
Not really, #59 fixed that all downloaded charts are present in the `.helm` directory at the end of the Tilt process. :)

**Comment by KOGI** [comment]
I have some ideas about offline mode, but haven't had a chance to try any of them out, yet. 😞

**Comment by robertmarsal** [comment]
To add to this issue, the update is also happening when running `tilt down` which adds an unnecessary delay to stopping the services
